docs/source/torchao_vllm_integration.md

```diff
@@ -0,0 +1,398 @@
+(torchao_vllm_integration)=
+# TorchAO Integration with VLLM: Architecture and Usage Guide
```

Comment:
Maybe drop TorchAO from the title? It's a bit long in the side bar now compared to the other pages
<img width="325" alt="Screenshot 2025-06-11 at 7 03 37 PM" src="https://github.com/user-attachments/assets/054ab95a-c6c2-4cd8-aca8-6298c7ff2682" />